libcxxabi/include/cxxabi.h

```diff
@@ -77,6 +77,9 @@ extern _LIBCXXABI_FUNC_VIS void __cxa_end_cleanup();
 #endif
 extern _LIBCXXABI_FUNC_VIS std::type_info *__cxa_current_exception_type();
 
+// GNU extension
+extern _LIBCXXABI_FUNC_VIS _LIBCXXABI_NORETURN void __cxa_call_terminate(void*) throw();
```

Comment:
I think this must be added to `libcxxabi/lib/itanium-exceptions.exp`. This will then automatically get it re-exported from libc++ when needed. That would break the ABI list, which you then need to fix.